### PR TITLE
PT-766: Fastlane secrets setup

### PR DESCRIPTION
**Pull request by amlcurran**
This is an initial PR which creates a project that can be used for the fastlane secrets plugin.

This just adds a basic iOS project with basic plugin infrastructure, nothing more. This will make the actual plugin easier to review!

### CODE REVIEW

File: Fastlane Secrets/Fastlane Secrets/AppDelegate.swift [hunk 1]
```diff
@@ -0,0 +1,46 @@
+//
```
Comment (gbasile) [inline Fastlane Secrets/Fastlane Secrets/AppDelegate.swift]: We should not have banner header in our source code

File: Fastlane Secrets/Fastlane Secrets/Base.lproj/Main.storyboard [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (FabiaS) [inline Fastlane Secrets/Fastlane Secrets/Base.lproj/Main.storyboard]: `Main.storyboard` 🙀

Comment (amlcurran) [reply Fastlane Secrets/Fastlane Secrets/Base.lproj/Main.storyboard]: I did delete this but in a later commit, let me see if I can chuck it out here

**Comment by amlcurran** [comment]
@gbasile @FabiaS did your requested changes!

File: Fastlane Secrets/Fastlane Secrets/AppDelegate.swift [hunk 3]
```diff
@@ -1,46 +1,29 @@
...
+        let viewController = UIViewController()
+        viewController.view.backgroundColor = .white
+        let label = UILabel()
+        viewController.view.addSubview(label)
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.leadingAnchor.constraint(equalTo: viewController.view.safeAreaLayoutGuide.leadingAnchor).isActive = true
+        label.trailingAnchor.constraint(equalTo: viewController.view.safeAreaLayoutGuide.trailingAnchor).isActive = true
+        label.topAnchor.constraint(equalTo: viewController.view.safeAreaLayoutGuide.topAnchor).isActive = true
+
+        label.text = "Secret key will go here!"
```
Comment (FabiaS) [inline Fastlane Secrets/Fastlane Secrets/AppDelegate.swift]: 😆 secret...

**Pull request merged**